src/consts.js

```diff
@@ -108,6 +108,18 @@ const DATASET_OUTPUT_FIELDS = [
     { key: 'itemsFileUrls', label: 'Items file URLs', type: 'string' },
 ];
 
+const KEY_VALUE_STORE_SAMPLE = {
+    keyValueStore: {
+        id: '98e7lEimuGBDvFfcM',
+        name: 'zapier-test',
```

Comment:
If this is shown in the UI then I would use name `my-store` or `my-key-value-store` rather ten `xxx-test`